packages/components/src/form-token-field/index.tsx

```diff
@@ -172,7 +172,7 @@ export function FormTokenField( props: FormTokenFieldProps ) {
 		if ( event.defaultPrevented ) {
```

Comment:
Unfortunately I've noticed some cross-browser differences — in Mac Chrome, the `event.key` fix is insufficient when <kbd>Enter</kbd> is involved (the other keys, and other Mac browsers, seem to be fine).

https://user-images.githubusercontent.com/555336/200654761-fc674d04-fab9-4a7c-aa50-6037dd0bdd94.mp4

☝️ Mac Chrome

I think we'll need to add a `isComposing` check:

```suggestion
		if ( event.defaultPrevented || event.nativeEvent.isComposing ) {
```

I guess technically we could just add the check in the `case 'Enter'` branch, but I think it would be safer to check it for all cases. In my experience there are some weird IME-related differences between OSes and browsers 😅 (Linux too!)

Reply:
Thanks for the test!
I think adding `isComposing` makes sense.

I am concerned that the same follow-up needs to be added for the corrections made in [AutocompleteUI](https://github.com/WordPress/gutenberg/issues/45496).